notify/receivers_test.go

```diff
@@ -113,13 +113,13 @@ func TestBuildReceiverConfiguration(t *testing.T) {
 	})
 	t.Run("should accept empty config", func(t *testing.T) {
 		recCfg := &APIReceiver{ConfigReceiver: ConfigReceiver{Name: "test-receiver"}}
-		parsed, err := BuildReceiverConfiguration(context.Background(), recCfg, decrypt)
+		parsed, err := BuildReceiverConfiguration(context.Background(), recCfg, DecodeSecretsFromBase64, decrypt)
 		require.NoError(t, err)
 		require.Equal(t, recCfg.Name, parsed.Name)
 	})
 	t.Run("should support non-base64-encoded secrets", func(t *testing.T) {
 		recCfg := &APIReceiver{ConfigReceiver: ConfigReceiver{Name: "test-receiver"}}
-		invalidBase64 := "not encoded"
+		invalidBase64 := "test"
```

Comment:
The string "test" is valid base64, the test will fail if the `DecodeSecretsFromBase64` decoding function is used.